src/Administration/Resources/app/administration/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./src/core/shopware.js",
   "types": "./types/index.d.ts",
   "scripts": {
-    "build": "export VITE_MODE=production ADMIN_VITE=$(jq -r '.ADMIN_VITE' \"$PROJECT_ROOT/var/config_js_features.json\") && [ \"$ADMIN_VITE\" = true ] && npm run vite build && ts-node -T build/plugins.vite.ts || mode=production node --no-deprecation ./node_modules/webpack/bin/webpack.js --config webpack.config.js",
+    "build": "export VITE_MODE=production ADMIN_VITE=$(jq -r '.ADMIN_VITE' \"$PROJECT_ROOT/var/config_js_features.json\") && [ \"$ADMIN_VITE\" = true ] && [ \"$SHOPWARE_ADMIN_BUILD_ONLY_EXTENSIONS\" != \"1\" ] && npm run vite build && ts-node -T build/plugins.vite.ts || ts-node -T build/plugins.vite.ts || mode=production node --no-deprecation ./node_modules/webpack/bin/webpack.js --config webpack.config.js",
```

Comment:
This is starting to get big. Should it be a .ts script?

Reply:
Next week we can remove most of it when trunk becomes unstable 💙